README.md

```diff
@@ -177,14 +177,14 @@ rm /root/autoEngageFailover/engage.lock
 mv cronRunDbUpdate.sh /etc/cron.hourly   
 
 # Below server cert change no longer needed as cert contains dual SAN
-# This means a server restart is no longer needed. 
-#echo Change server certificate and key back for goc.dl.ac.uk
-#ln -sf /etc/pki/tls/private/goc.dl.ac.uk.key.pem /etc/pki/tls/private/hostkey.pem
-#ln -sf /etc/grid-security/goc.dl.ac.uk.cert.pem /etc/grid-security/hostcert.pem
+# This means a server restart is no longer needed.
+#echo Change server certificate and key back for gocdb.hartree.stfc.ac.uk
+#ln -sf /etc/pki/tls/private/gocdb.hartree.stfc.ac.uk.key.pem /etc/pki/tls/private/hostkey.pem
+#ln -sf /etc/grid-security/gocdb.hartree.stfc.ac.uk.cert.pem /etc/grid-security/hostcert.pem
 #service httpd restart
 #service gocdb-autofailover start
 #service gocdb-autofailover status
```

Comment:
Should this be deleted rather than updated if it is not longer required?

Reply:
I wrestled with this, and decided that was a separate issue 😛 #10